### PR TITLE
F25 dev fix bootloader bootpart

### PR DESCRIPTION
**Pull request by jkonecny12**
This is PR https://github.com/rhinstaller/anaconda/pull/706 for the f25 and master.

### CODE REVIEW

**Comment by rvykydal** [comment]
To the extent I understand the domain, the patch looks OK to me.

**Comment by vpodzime** [comment]
> To the extent I understand the domain, the patch looks OK to me.

Same here.

**Comment by jkonecny12** [comment]
jenkins, test this please

File: pyanaconda/kickstart.py [hunk 1]
```diff
@@ -386,26 +409,39 @@ def execute(self, storage, ksdata, instClass):
...
+            # In the first run, the result is used for other partitioning but
+            # the second will be used.
+            try:
+                boot_dev = storage.mountpoints["/boot"]
+            except KeyError:
+                log.debug("Bootloader: /boot partition is not present, dry run %s", dry_run)
+            else:
+                boot_drive = ""
+                # Find disk ancestor
+                for ancestor in boot_dev.ancestors:
```
Comment (dwlehman) [inline pyanaconda/kickstart.py]: There is also `boot_dev.disks`.

Comment (jkonecny12) [inline pyanaconda/kickstart.py]: Have the ``boot_dev.disks`` some advantages over ``ancestors``?

Comment (dwlehman) [inline pyanaconda/kickstart.py]: They have the advantage of being, by definition, disks, which would simplify your code a bit.

Comment (jkonecny12) [inline pyanaconda/kickstart.py]: Ahh ok, thank you.

**Comment by jkonecny12** [comment]
Updated, thank you @dwlehman for the advice.

Because it's only this small change I'm going to merge it now.

**Pull request merged**